### ISSUE TITLE
Setting a location via (at least) Immich Web sets the coordinates, but not the country/state/city

### ISSUE THREAD

**Issue by freak12techno** [immich-app/immich#12201]
### The bug

I have a usecase: I want to select all of the photos that have no location, and set the location for these manually.
I can now search for images without country, now with https://github.com/immich-app/immich/discussions/8362 merged, but this still doesn't work correctly.

This is what I do in Immich Web (haven't tried doing it on mobile, so not sure if that'll work the same way there):
1. I create a new search, with country = Unknown
2. I get a list of images without location
3. I open the image, press "Add a location", choose a location on the map and save it
4. I refresh the search page
5. I see this image still being present, when clicking on it, it shows "Add a location", although if clicking on it, it shows the location I just added, and if I see the `/api/assets/<asset ID>` endpoint response for this image, it would show that it has latitude and longitude in exifInfo, but not the country/state/city fields, all of these are null (they are not null if I import an image that has all of these enabled)

So basically my goal is to make this list empty, but adding the location there doesn't remove it from this list and doesn't set the region fields correctly, only adds the latitude and longitude.

Can the endpoint to update location be updated so it'd also prefill these fields?

### The OS that Immich Server is running on

Unraid + docker-compose

### Version of Immich Server

v1.113.0

### Version of Immich Mobile App

- 

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
The default one
```

### Your .env content

```Shell
Pretty much the default.
```

### Reproduction steps

See above.

**Comment by bo0tzz** [comment]
The named fields are filled by the reverse geocoding portion of the metadata extraction job. If you're editing a bunch of assets at once, it may take a bit for any one asset to get its turn to be processed.
